include/networkit/centrality/KadabraBetweenness.hpp

```diff
@@ -175,11 +175,7 @@ class KadabraBetweenness : public Algorithm {
 
     count maxAllocatedFrames() const {
         assureFinished();
-        count maxNFrames = 0;
-        for (auto nFrames : maxFrames) {
-            maxNFrames = std::max(nFrames, maxNFrames);
-        }
-        return maxNFrames;
+        return *std::max_element(maxFrames.begin(), maxFrames.end());
```

Comment:
This faults if maxFrames is empty.

Reply:
`maxFrames` is resized with `omp_get_max_threads()` before each execution, it should never be empty right?

Reply:
Where is that done? In the constructor?

Reply:
It's done in `init()` (always called by `run()`)
https://github.com/networkit/networkit/blob/eec604c774d0bf98ce6c5829f3eb73ac0358d16e/networkit/cpp/centrality/KadabraBetweenness.cpp#L280

Here we are guaranteed that `run()` was called by `assureFinished()`.